### ISSUE TITLE
Update Tag A11y docs

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#5457]
### Related page(s)

https://www.saltdesignsystem.com/salt/components/tag/accessibility

### Kind of issue

Missing information

### Description

Update A11y docs and clarify WCAG use of color 1.4.1 considerations.

### Browser

- [x] Chrome
- [x] Safari
- [x] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.
